File: SSIS/SSIS/Models/Util Models/RetrievalList.cs
using SSIS.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SSIS.View_Models
{
    public class RetrievalList
    {
        public Item RetrievedItem { get; set; }
        public int Needed { get; set; }
        public int Retrieved { get; set; }
        public List<Retrieval> Retrievals { get; set; }
      

    }
}

File: Android/app/src/main/java/com/arun/api/Model/RetrievalList.java
package com.arun.api.Model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class RetrievalList implements Serializable
{
    @SerializedName("RetrievedItem")
     String RetrievedItem;
    @SerializedName("Needed")
     int Needed ;
    @SerializedName("Retrieved")
     int Retrieved ;
    @SerializedName("Retrievals")
     List<Retrieval> Retrievals ;

    public String getRetrievedItem() {
        return RetrievedItem;
    }

    public void setRetrievedItem(String retrievedItem) {
        RetrievedItem = retrievedItem;
    }

    public int getNeeded() {
        return Needed;
    }

    public void setNeeded(int needed) {
        Needed = needed;
    }

    public int getRetrieved() {
        return Retrieved;
    }

    public void setRetrieved(int retrieved) {
        Retrieved = retrieved;
    }

    public List<Retrieval> getRetrievals() {
        return Retrievals;
    }

    public void setRetrievals(List<Retrieval> retrievals) {
        Retrievals = retrievals;
    }
}
